### DISCUSSION TITLE
Promitor on AKS deployment - Privilege escalation container is not allowed error

### DISCUSSION THREAD

**Discussion by JanMwork** [tomkerkhove/promitor#2178] [Q&A] [answered]
Hello,

I am trying to deploy Promitor into the AKS environment, where runAsRoot is not allowed.

Hence, my YAML contains: 

spec:
  containers:
  securityContext:
    runAsNonRoot: true

When I run "kubectl apply -f promitor.yaml" the following error is displayed :

[azurepolicy-XXX] Privilege escalation container is not allowed: mypod.

Does it mean it is impossible to run Promitor as NonRoot (I haven't found such a statement in the documentation)?

Thank you.
Jan

**Comment by tomkerkhove** [comment, marked as answer]
I have not tested it to run as non-root so feel free to open an issue for this.

**Reply by JanMwork** [reply]
Done, thank you @tomkerkhove !

**Reply by tomkerkhove** [reply]
https://github.com/tomkerkhove/promitor/issues/2182
